.editorconfig

```diff
@@ -0,0 +1,158 @@
+# EditorConfig is awesome:http://EditorConfig.org
+# .NET coding convention settings for EditorConfig
+# https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
+#
+# The .editorconfig comes from Roslyn and CoreFX repositories:
+# https://github.com/dotnet/corefx/blob/master/.editorconfig
+# https://github.com/dotnet/roslyn/blob/master/.editorconfig
+
+# Top-most EditorConfig file
+root = true
+
+# Don't use tabs for indentation.
+[*]
+indent_style = space
+# (Please don't specify an indent_size here; that has too many unintended consequences.)
+
+# Code files
+[*.{cs,csx,vb,vbx}]
```

Comment:
We don't have any vb code, but we do have cpp.

Reply:
Fixed.